.travis.yml

```diff
@@ -34,8 +34,8 @@ jobs:
           install:
               - pip3 install --upgrade pip
               - pip3 install --progress-bar off poetry
-              - python3 -m poetry --no-interaction install
-              - python3 -m poetry --no-interaction build
+              - python3 -m poetry install --no-interaction
```

Comment:
Why? What is the purpose of these changes?

Reply:
Poetry 1.0 wants flags (e.g. `--no-interaction`) to be after the subcommand

https://travis-ci.org/github/datadvance/DjangoChannelsGraphqlWs/jobs/626173702#L404

![image](https://user-images.githubusercontent.com/26336/78461626-6334e300-7690-11ea-9e4d-31d437884a8b.png)